ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-ref.yaml

```diff
@@ -18,5 +18,6 @@ ref:
   - namespace: test-credentials
     name: telco-bastion-userpw
     mount_path: /var/run/ssh-pass
+  timeout: 3h
```

Comment:
3h seems hugely long, but [in the rehearsal it took 1h28m][1].  Can we get some notes in the commit message explaining why we feel like 3h is reasonable here?

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/24048/rehearse-24048-periodic-ci-openshift-release-master-nightly-4.10-e2e-telco5g/1472613242286116864#1:build-log.txt%3A13

Reply:
done